cnc_ctrl_v1/System.h

```diff
@@ -71,6 +71,7 @@ extern system_t sys;
 extern Axis leftAxis;
 extern Axis rightAxis;
 extern Axis zAxis;
+extern Axis axes[3];
```

Comment:
I don't know enough about c++, how is something like this stored?  How much memory does this array consume?  Is it only a reference to the existing axes or does it contain a duplicate instance?

Reply:
Good point... unfortunately I don't know either, so... I forced it to become an array of pointers to the real axes...